File: java/src/main/java/com/example/app/common/vo/base/ResponseCode.java
package com.example.app.common.vo.base;

/**
 * 响应状态码
 */
public enum ResponseCode {
    SUCCESS("200", "成功"),
    ERROR("500", "服务器内部未知错误"),
    ERROR_VALID("00001001", "业务异常"),
    PARAM_VALID("00001002", "参数异常");
    private String code;
    private String message;

    private ResponseCode(String code, String message) {
        this.code = code;
        this.message = message;
    }

    public String getCode() {
        return this.code;
    }

    public String getMessage() {
        return this.message;
    }
}

File: php/ResponseCode.php
<?php
namespace App\Common;

/**
 * 返回码定义
 */
class ResponseCode{
    // 成功
    const SUCCESS = 1;
    const SYS_ERROR = 0;


    const USERNAME_ERROR = 2001; //用户名不合规
    const PASSWORD_ERROR = 2002; //密码不合规

}
